Scripts/script-DemistoDeleteIncident.yml

```diff
@@ -0,0 +1,35 @@
+commonfields:
+  id: DemistoDeleteIncident
+  version: -1
+name: DemistoDeleteIncident
+script: |-
+  var body = {
+      ids: [args.id],
+      all: false,
+      filter: {}
+  };
+
+  var res = executeCommand('demisto-api-post', {url: 'incident/batchDelete', body: body});
+  if (res[0].Type == 4) {
```

Comment:
use constant and not 4 (change in all places)

Reply:
changed to isError (much better, thanks)